lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -1621,7 +1621,9 @@ func (d *zfs) commonVolumeRules() map[string]func(value string) error {
 		"zfs.use_refquota": validate.Optional(validate.IsBool),
 		// lxdmeta:generate(entities=storage-zfs; group=volume-conf; key=zfs.delegate)
 		// This option controls whether to delegate the ZFS dataset and anything underneath it to the
-		// container or containers that use it. This allows using the `zfs` command in the container.
+		// container or containers that use it. When used in conjunction with
+		// {config:option}`instance-security:security.nesting`, this allows
+		// using the `zfs` command in the container.",
```

Comment:
```suggestion
		// using the `zfs` command in the container.
```

Reply:
Why the `",` at the end of the line? It renders like this in [the preview](https://canonical-ubuntu-documentation-library--14629.com.readthedocs.build/lxd/en/14629/reference/storage_zfs/#storage-zfs-volume-conf:zfs.delegate):

![image](https://github.com/user-attachments/assets/2fe7ccf4-cf89-4bc7-94d6-37fae982f5a7)

Reply:
Copy-n-pasto, now fixed, thanks for catching!